hooks/hookexecution/execution_test.go

```diff
@@ -137,6 +143,9 @@ func TestHandleModuleActivitiesNoBidderRequestPayload(t *testing.T) {
 			activityControl := privacy.NewActivityControl(test.privacyConfig)
 			newPayload := handleModuleActivities(test.hookCode, activityControl, test.inPayloadData)
 			assert.Equal(t, test.expectedPayloadData, newPayload)
+			//check input payload didn't change
+			origInPayloadData := test.inPayloadData
+			assert.Equal(t, origInPayloadData, test.inPayloadData)
```

Comment:
I think you need to move the line `origInPayloadData := test.inPayloadData` above the call to `handleModuleActivities`.